website/content/docs/api-gateway.mdx

```diff
@@ -0,0 +1,192 @@
+
+## Introduction
+
+Consul API Gateway is an implementation of the Kubernetes Gateway [API Specification](https://gateway-api.sigs.k8s.io/). This specification defines a set of custom resource definitions (CRD) that can create logical gateways and routes based on the path or protocol of a client request. The Consul API gateway solves for two primary use cases:
+
+* **Controlling access at the point entry** - The Consul API Gateway allows users to set the protocols of external connection requests and provide clients with TLS certificates from trusted providers (e.g. VeriSign, Let’sEncrypt). 
+* **Simplifying traffic management** - The Consul API Gateway can be used to load balance requests across services and route traffic to the appropriate service based on matching one or more criteria such as hostname, path, header presence or value, and HTTP Method type (e.g. Get, Post, Patch).
+
+## Requirements
+
+Your datacenter must meet the following requirements prior to configuring the Consul API Gateway:
+
+- A Kubernetes cluster must be running
+- Consul 1.11.2+
+
+## Installation
+
+1. Create a values file for your Consul server agents that contains the following parameters:
+
+  <CodeBlockConfig hideClipboard filename="values.yaml">
+
+```yaml
+global:
+  name: consul
+  image: 'hashicorp/consul:1.11.2'
+  tls:
+    enabled: true
+connectInject:
+  enabled: true
+controller:
+  enabled: true
+```
+
+  </CodeBlockConfig>
+
+1. Install Consul API Gateway using the standard Consul Helm chart and specify the custom values file.
+
+```shell-session
+$ helm install consul hashicorp/consul --version 0.39.0 --values values.yaml
+```
+
+The following components will be installed: - Gateway controller - CRDs required by the Kubernetes Gateway API specification - `kustomize` manifests for completing the installation
+
+1. After `helm` installs Consul API Gateway packages, issue the following commands to apply the API gateway to your Kubernetes cluster.
+
+```shell-session
+$ kubectl apply --kustomize="github.com/hashicorp/consul-api-gateway/config/crd?ref=v0.1.0-beta"
+$ kubectl apply --kustomize="github.com/hashicorp/consul-api-gateway/config?ref=v0.1.0-beta"
+```
+
+## Usage
+
+1. Verify that the [requirements](#requirements) have been met.
+1. Verify that the Consul API Gateway software has been installed and applied (see [Installation](#installation)).
+1. Configure the gateway, listener(s), and route(s) as described in [Configuration](#configuration)
+1. Issue the `kubectl apply` command to implement the configurations, e.g.:
+
+```shell-session
+$ kubectl apply --values gateway-configuration.yaml
+```
+
+### Using the Consul API Gateway Binary
```

Comment:
I would just consider dropping this section for now. We really only support tight kubernetes integration at the moment in which case you'll need to jump through the hoops of manually setting up a kubernetes authentication context for the binary and make sure the host you're running it on is routable from all potential kubernetes worker nodes. No one should really be running that in a production setup.

Once we have support for other runtimes (i.e. potentially something like a standalone runtime), invoking this directly or running it as part of a kubernetes installation may make sense, but for now, it doesn't.

Reply:
Commented out